draft-ietf-lamps-pq-composite-kem.md

```diff
@@ -673,27 +676,29 @@ EDNOTE: these domain separators are based on the prototyping OIDs assigned on th
 
 ## RSA-OAEP Parameters {#sect-rsaoaep-params}
 
-Use of RSA-OAEP [RFC3560] within `id-MLKEM512-RSA2048` and `id-MLKEM512-RSA3072` requires additional specification.
+Use of RSA-OAEP [RFC8017] within `id-MLKEM512-RSA2048` and `id-MLKEM512-RSA3072` requires additional specification.
 
 First, a quick note on the choice of RSA-OAEP as the supported RSA encryption primitive. RSA-KEM [RFC5990] is more straightforward to work with, but it has fairly limited adoption and therefore is of limited backwards compatibility value. Also, while RSA-PKCS#1v1.5 [RFC8017] is still everywhere, but hard to make secure and no longer FIPS-approved as of the end of 2023 [SP800-131Ar2], so it is of limited forwards value. This leaves RSA-OAEP [RFC3560] as the remaining choice.
 
 The RSA component keys MUST be generated at the 2048-bit and 3072-bit security levels respectively.
 
 As with the other composite KEM algorithms, when `id-MLKEM512-RSA2048` or `id-MLKEM512-RSA3072` is used in an AlgorithmIdentifier, the parameters MUST be absent. The RSA-OAEP SHALL be instantiated with the following hard-coded parameters which are the same for both the 2048 and 3072 bit security levels.
 
-| RSA-OAEP Parameter      | Value                            |
-| ---------------------- | ---------------                  |
-| hashFunc                  | id-sha2-256                   |
-| maskGenFunc               | mgf1SHA256Identifier          |
-| pSourceFunc           | DEFAULT pSpecifiedEmptyIdentifier |
-| ss_len                    | 256 bits                      |
+| RSAES-OAEP-params           | Value                       |
+| ----------------------      | ---------------             |
+| hashAlgorithm               | id-sha2-256                 |
+| maskGenAlgorithm            | mgf1SHA256Identifier        |
+| pSourceAlgorithm            | pSpecifiedEmpty             |
+| ss_len                      | 256 bits                    |
 {: #rsa-oaep-params title="RSA-OAEP Parameters"}
 
 where:
 
 * `id-sha256` is defined in [RFC8017].
-* `mgf1SHA256Identifier` is defined in [RFC4055].
-* `pSpecifiedEmptyIdentifier` is defined in [RFC3560]
+* `mgf1SHA256Identifier` is defined in [RFC4055], which refers to the MFG1 function defined in [RFC8017] appendix B.2.1.
+* `pSpecifiedEmpty` is defined in [RFC8017] to indicate that the empty string is used for the label.
+
+EDNOTE: we could determine the mask length for each parameter set given in this document. According to 8017, it will be the length of `k - hLen - 1`, where `k` is the size of the RSA modulus. I'm not sure that we strictly need to calculate and list these.
```

Comment:
Turn this into a real note.
"... since the mask length will be constant for each composite algorithm, implementations could pre-compute and hard-code the mask length value."